tests/josegonzalez/Queuesadilla/Engine/MongoEngineTest.php

```diff
@@ -0,0 +1,397 @@
+<?php
+namespace josegonzalez\Queuesadilla\Engine;
+
+use josegonzalez\Queuesadilla\Engine\MongoEngine;
+use MongoDB\BSON\ObjectId;
+use Psr\Log\NullLogger;
+
+class MongoEngineTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @var \josegonzalez\Queuesadilla\Engine\MongoEngine
+     */
+    protected $Engine;
+    /**
+     * @var NullLogger
+     */
+    private $Logger;
+
+    /**
+     * @inheritDoc
+     */
+    public function setUp(): void
```

Comment:
This assumes a modern PHP, while the package still supports 5.6.

Reply:
removed.